src/ngraph/output_vector.hpp

```diff
@@ -1,5 +1,5 @@
 //*****************************************************************************
-// Copyright 2017-2019 Intel Corporation
+// Copyright 2017-2020 Intel Corporation
```

Comment:
If you applied copyright changes in an independent commit, could we please revert it and create a new PR for that? It would be useful to have those changes so that we don't lose them if the MKL-DNN code in this PR has to be reverted and it.